client/src/Components/NewsCard.js

```diff
@@ -10,29 +10,41 @@ import CardContent from "@material-ui/core/CardContent";
 import CardMedia from "@material-ui/core/CardMedia";
 import Collapse from "@material-ui/core/Collapse";
 import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
-import Grid from "@material-ui/core/Grid";
 import IconButton from "@material-ui/core/IconButton";
 import Pause from "@material-ui/icons/NotInterested";
 import TimeAgo from "@material-ui/icons/AccessTime";
+
 import Typography from "@material-ui/core/Typography";
 
+const calculateArticleAge = require("../Helpers/calculateArticleAge");
+
 const styles = theme => ({
-  newsCard: {
-    marginBottom: "15px",
+  card: {
+    marginBottom: 20,
   },
   media: {
-    width: "93px",
-    height: "68px",
-    marginTop: 20,
-    marginLeft: 20,
+    height: 0,
+    paddingTop: "56.25%", // 16:9
   },
   title: {
-    fontSize: 13.5,
-    fontWeight: 500,
     color: "textPrimary",
-    textTransform: "uppercase",
-    paddingLeft: 0,
-    fontFamily: "Source Sans Pro",
+    fontFamily: "Avenir",
```

Comment:
@mirhamasala Is the change in font related to the new design?

Reply:
Yes.

I'd like to set this change site-wide. But I need to take the time to see how that ties in with Material UI.